keyboards/reviung/reviung46/keyboard.json

```diff
@@ -0,0 +1,113 @@
+{
+    "keyboard_name": "reviung46",
+    "manufacturer": "gtips",
+    "url": "https://github.com/gtips/reviung/tree/master/reviung46/",
+    "maintainer": "gtips",
+    "usb": {
+        "vid": "0x4E95",
+        "pid": "0x4E19",
+        "device_version": "0.0.1"
+    },
+    "rgblight": {
+        "hue_steps": 16,
+        "saturation_steps": 16,
+        "brightness_steps": 16,
+        "led_count": 10,
+        "sleep": true,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "D3"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": true
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "matrix_pins": {
+        "cols": ["B5", "C6", "D4", "D0", "D1", "D2"],
+        "rows": ["B6", "B2", "B3", "B1", "F7", "F6", "F5", "F4"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "atmega32u4",
+    "bootloader": "caterina",
```

Comment:
```suggestion
    "development_board": "promicro",
```

Reply:
Thanks.
I fixed it.

Reply:
Doesn't appear so, did you forget to push this change?

Reply:
I fixed it and pushed it, but it didn't show up. 
Did I make a mistake?

Reply:
The lines suggested for removal have not been removed. They are not needed when using `development_board`.

Reply:
Thanks.
I fixed it.

Reply:
I fixed it.
Request review.